.devcontainer/postAttach.sh

```diff
@@ -1,6 +1,17 @@
 #!/usr/bin/env bash
 set -eu
 
+# The global gitconfig gets copied into the container:
+#
+# https://code.visualstudio.com/docs/remote/containers#_sharing-git-credentials-with-your-container
+#
+# and may have a hooksPath setting that is incompatible with pre-commit.
+#
+# https://github.com/pre-commit/pre-commit/issues/1198#issuecomment-547478570
+#
+# Remove it if present.
+git config --global --unset-all core.hooksPath
```

Comment:
I'm seeing an error message when trying to restart the devcontainer, that the `postAttach` command failed:

```console
Running the postAttachCommand from devcontainer.json...

[45015 ms] Start: Run in container: /bin/bash .devcontainer/postAttach.sh
[46517 ms] postAttachCommand failed with exit code 5. Skipping any further user-provided commands.
```

Can confirm that `pre-commit` is not initialized, nor are the `tests/cypress` NPM packages installed or initialized.